docs/source/release_notes.rst

```diff
@@ -104,6 +137,26 @@ Requirements
 The IBM z/OS core collection has several dependencies, please review the `z/OS core support matrix`_ to understand both the
 controller and z/OS managed node dependencies.
 
+Known Issues
+------------
+
+- ``zos_job_submit`` - when setting 'location' to 'LOCAL' and not specifying the from and to encoding, the modules defaults are not read leaving the file in its original encoding; explicitly set the encodings instead of relying on the default.
+- ``zos_job_submit`` - when submitting JCL, the response value returned for **byte_count** is incorrect.
+
+- ``zos_job_submit``, ``zos_job_output``, ``zos_operator_action_query`` - encounters UTF-8 decoding errors when interacting with results that contain non-printable UTF-8 characters in the response. This has been addressed in this release and corrected with **ZOAU version 1.2.5.6** or later.
```

Comment:
This reads a bit awkward since our minimum is now zoau 1.3.0 but beyond that, this was fixed in zoau 1.3 anyhow but that introduced a new issue in the same code/space, that is a jsondecode issue now. So this sentence should be updated to point out that known issue, so replace this sentence:

```
- ``zos_job_submit``, ``zos_job_output``, ``zos_operator_action_query`` - encounters UTF-8 decoding errors when interacting with results that contain non-printable UTF-8 characters in the response. This has been addressed in this release and corrected with **ZOAU version 1.2.5.6** or later.
```

```
- ``zos_job_submit``, ``zos_job_output``, ``zos_operator_action_query`` - encounters JSON decoding (DecodeError, TypeError, KeyError) errors when interacting with results that contain non-printable UTF-8 characters in the response. This will be addressed in **ZOAU version 1.3.2** and later.
```

Reply:
looks good

Reply:
^ i think this comment applies to the release notes for v1.10.0 (line 125). This section looks like it's part of v1.9.1 (beginning at line 128), for which ZOAU 1.3.x would be incorrect.